dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/bnd.bnd

```diff
@@ -1,5 +1,5 @@
 #*******************************************************************************
-# Copyright (c) 2017 IBM Corporation and others.
+# Copyright (c) 2018 IBM Corporation and others.
```

Comment:
This isn't correct. Unless the file is entirely rewritten, the format is "initial year, modified year" at most. Even just "initial year" is more correct than removing the initial year.